patterns/creational/builder.md

```diff
@@ -106,3 +114,4 @@ as well as the `FooBuilder::new().a().b().build()` style.
 - [Constructor pattern](../idioms/ctor.md) for when construction is simpler.
 - [Builder pattern (wikipedia)](https://en.wikipedia.org/wiki/Builder_pattern)
 - [Construction of complex values](https://web.archive.org/web/20210104103000/https://rust-lang.github.io/api-guidelines/type-safety.html#c-builder)
+- [Motivation to add `builder` functions struct to the struct](https://matklad.github.io/2020/08/12/who-builds-the-builder.html)
```

Comment:
```suggestion
```

I think it's not necessary to link that short blogpost here. You've already said the `builder` method is for easier discovery in the comment of that method.